tests/morestripe.test/runit

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
Add this test: writes lots of blobs when blobstripe is disabled, blobstripe it, write more, make sure all blobs are correct